taxi/models.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from taxi_service.settings import AUTH_USER_MODEL
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    county = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True)
+
+    class Meta:
+        verbose_name_plural = "drivers"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ManyToManyField(AUTH_USER_MODEL)

```

Comment:
add related name for Car model)
dont forget about [chechlist.md](https://github.com/mate-academy/py-taxi-service-startproject/blob/master/checklist.md)